.github/workflows/platform-executables.yml

```diff
@@ -54,3 +54,36 @@ jobs:
           ASSET_NAME: ${{ matrix.asset_name }}
           TAG_NAME: ${{ needs.get-tag.outputs.TAG_NAME }}
           REPO_NAME: twilio/twilio-cli
+  notify-complete-success:
+    needs: [ platform-executables-release ]
+    name: Notify Release Completed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEB_HOOK }}
+          SLACK_COLOR: "#36a64f"
+          SLACK_USERNAME: CLI Release Bot
+          SLACK_ICON_EMOJI: ":ship:"
+          SLACK_TITLE: 'Twilio CLI'
+          SLACK_MESSAGE: 'Platform Executable Release Completed'
+          MSG_MINIMAL: actions url
```

Comment:
Are we planning on sending out separate notifications for every type of release? Do we need this granularity? 

It makes more sense to me to just send out `Successful CLI Release` incase all the workflows (self and children) of `twilio-cli` have succeeded (But for failure in #alert-dev-interfaces should be at each workflow level).

Reply:
Actually the problem here is docker, platform executables and homebrew release are run in parallel. I will be able to send one success notification after docker and platform executables are done, but homebrew workflow run in another repo. I can reduce it to send two success notifications. But I was not able to find a way to send only one notification. Please let me know if you think otherwise.

Reply:
Updated with single successful slack notification.